localisation/00_wc_vanilla_replacements.csv

```diff
@@ -227,10 +227,10 @@ HF_211_DESC;Lying in the crib, I hear a slithering presence drawing closer and c
-EVTDESCHF23001;[From.GetTitledFirstName] was sacrificed to [FromFrom.Religion.GetHighGodName]! Accounts describe how [From.GetSheHe] was forcefully dragged screaming on top of a temple and torn apart by mad priests.;;;;;;;;;;;;;x
-EVTDESC_SSI_060_B;[From.GetTitledName] was sacrified to [FromFrom.Religion.GetHighGodName] by [FromFrom.GetFirstName]'s priests! After bringing [From.GetHerHim] to one of their temples, they placed [From.GetFirstName] on an altar and sliced [From.GetHerHim] open with a ceremonial dagger. They then tore out [From.GetHerHis] still beating heart to the cheers of the assembled crowd.;;;;;;;;;;;;;x
-EVTDESC_SSI_072;Your priests have sacrificed [From.GetTitledName] to [Root.Religion.GetHighGodName]! After bringing [From.GetHerHim] to one of our temples, the priests placed [From.GetFirstName] on an altar and sliced [From.GetHerHim] open with a ceremonial dagger. They then tore out [From.GetHerHis] still beating heart to the cheers of the assembled crowd.;;;;;;;;;;;;;x
-EVTOPTA_SSI_072;[Root.Religion.GetHighGodName] will be pleased.;;;;;;;;;;;;;x
+EVTDESCHF23001;[target_localization_sacrifice.GetTitledFirstName] was sacrificed to [FromFrom.Religion.GetHighGodName]! Accounts describe how [target_localization_sacrifice.GetSheHe] was forcefully dragged screaming on top of a temple and torn apart by mad priests.;;;;;;;;;;;;;x
+EVTDESC_SSI_060_B;[From.GetTitledName] was sacrified to [FromFrom.Religion.GetHighGodName] by [FromFrom.GetFirstName]'s priests! After bringing [From.GetHerHim] to one of their [FromFrom.GetHouseOfWorship], they placed [From.GetFirstName] on an altar and sliced [From.GetHerHim] open with a ceremonial dagger. They then tore out [From.GetHerHis] still beating heart to the cheers of the assembled crowd.;;;;;;;;;;;;;x
```

Comment:
I'm not an expert, but `GetHouseOfWorship` is singular and it's incorrect to say `one of their <Singular Noun>`.

Reply:
You're right. Should add "s" after the <>.

Reply:
Better get rid of the construction and just use singular form. You can't be sure plural form will look like this.

Reply:
Fixed